### ISSUE TITLE
Incompatible with Firefox Quantum

### ISSUE THREAD

**Issue by lcc1211** [downthemall/downthemall-legacy#275]
### Your DownThemAll! and Browser Version and Language and Operating System

I don't have it installed right now, but my OS is Windows 10, English and 57.0.2 64 bit for Firefox
### Expected Behavior

I expected to be able to install it
### Actual Behavior

Message says it is not compatible.
### Steps To Reproduce
Install my version of Firefox.

### Additional info such as log extracts
*Please enable Diagnostic Logging in the Privacy tab of the DownThemAll! preferences and either copy/paste essential part of the log or attach the entire log. Please note that the log may include sentitive information which you have to remove prior to posting it!*
I can't do this since it isn't compatible.
![capture](https://user-images.githubusercontent.com/24764870/33833000-250ae2ca-de4c-11e7-9b25-a84241629a31.JPG)

**Comment by ladysham** [comment]
Yes, I have the same issue.   Sure miss the ease of using Download Them All!  Any possibility of an update?

**Comment by udo1toni** [comment]
See http://www.downthemall.net/delays/

Duplicate to #270

**Comment by avatar21** [comment]
Ahhh, I'm so lost without downthemalll ... I'm the type of fans that'll probably install Firefox just because of DownThemAll!

**Comment by jarredou** [comment]
update: [https://www.downthemall.net/lack-of-progress/](https://www.downthemall.net/lack-of-progress/)

**Comment by ladysham** [comment]
Thank you for the update.  I just sent you a donation.  This is a plugin 
that I used every day.  I will be glad when you have an opportunity to 
complete it.  Merry Christmas!

Warm Regards,

Kelly Ling

**Comment by jarredou** [comment]
@ladysham I'm not involved in DTA, just following its developement in the current context, and transmitting news from the official website for people who are waiting for some infos here ;)
